geode-core/src/main/java/org/apache/geode/management/internal/cli/i18n/CliStrings.java

```diff
@@ -2400,7 +2400,7 @@
   public static final String START_LOCATOR__RUN_MESSAGE = "Starting a Geode Locator in %1$s...";
   public static final String START_LOCATOR__CONNECT = "connect";
   public static final String START_LOCATOR__CONNECT__HELP =
-      "When connect is set to false , Gfsh does not automatically connect to the locator which is started using this command.";
+      "When connect is set to false or when locator is started with a security manager using --J=-Dgemfire.security-manager option, Gfsh does not automatically connect to the locator which is started using this command.";
```

Comment:
I thought the intent was that the user should see a message like:

`Security Manager is enabled - unable to auto-connect. Please connect with --user and --password` when they start a locator.

Reply:
I see that the ticket says about updating the help text and I didn't like adding this to help text of `connect` option anyways but I couldn't find a better place in help.

Will update the output showing that he/she is not connected.

Reply:
Done fixed both help text and command output as requested in the ticket.